packages/zwave-js/src/lib/commandclass/SceneActuatorConfigurationCC.ts

```diff
@@ -0,0 +1,422 @@
+			// to be different from the requested sceneId, and the client won't
+			// be able to determine the actual sceneId, because it's hidden in
+			// the return value
+			return;
+		}
+
+		const cc = new SceneActuatorConfigurationCCGet(this.driver, {
+			nodeId: this.endpoint.nodeId,
+			endpoint: this.endpoint.index,
+			sceneId: sceneId,
+		});
+		const response = await this.driver.sendCommand<SceneActuatorConfigurationCCReport>(
+			cc,
+			this.commandOptions,
+		);
+
+		if (!response) return;
+		if (response.sceneId === sceneId) {
+			return pick(response, ["level", "dimmingDuration"]);
+		} else {
+			// I don't think this can happen, if we block sceneId = 0
+			return;
+		}
+	}
+}
+
+@commandClass(CommandClasses["Scene Actuator Configuration"])
+@implementedVersion(1)
+export class SceneActuatorConfigurationCC extends CommandClass {
+	declare ccCommand: SceneActuatorConfigurationCommand;
+
+	// interview is omitted, because our only option would be
+	// to query all 255 possible sceneIds, which is a lot of traffic
+	// for little benefit
+}
+
+interface SceneActuatorConfigurationCCSetOptions extends CCCommandOptions {
+	sceneId: number;
+	dimmingDuration: Duration;
+	level?: number;
+}
+
+@CCCommand(SceneActuatorConfigurationCommand.Set)
+export class SceneActuatorConfigurationCCSet extends SceneActuatorConfigurationCC {
+	public constructor(
+		driver: Driver,
+		options:
+			| CommandClassDeserializationOptions
+			| SceneActuatorConfigurationCCSetOptions,
+	) {
+		super(driver, options);
+		if (gotDeserializationOptions(options)) {
+			// TODO: Deserialize payload
+			throw new ZWaveError(
+				`${this.constructor.name}: deserialization not implemented`,
+				ZWaveErrorCodes.Deserialization_NotImplemented,
+			);
+		} else {
+			if (options.sceneId < 1 || options.sceneId > 255) {
+				throw new ZWaveError(
+					`scene id ${options.sceneId} out of range [1..255]`,
+					ZWaveErrorCodes.Argument_Invalid,
+				);
+			}
+			this.sceneId = options.sceneId;
+			this.dimmingDuration = options.dimmingDuration;
+			this.level = options.level;
+		}
+	}
+
+	public sceneId: number;
+	public dimmingDuration: Duration;
+	public level?: number;
+
+	public serialize(): Buffer {
+		this.payload = Buffer.from([
+			this.sceneId,
+			this.dimmingDuration.serializeSet(),
+			this.level != undefined ? 0b1000_0000 : 0,
+			this.level ?? 0xff,
+		]);
+		return super.serialize();
+	}
+}
+
+@CCCommand(SceneActuatorConfigurationCommand.Report)
+export class SceneActuatorConfigurationCCReport extends SceneActuatorConfigurationCC {
+	public constructor(
+		driver: Driver,
+		options: CommandClassDeserializationOptions,
+	) {
+		super(driver, options);
+		validatePayload(this.payload.length >= 3);
+		this.sceneId = this.payload[0];
+
+		if (this.sceneId !== 0) {
+			this.level = this.payload[1];
+			this.dimmingDuration =
+				Duration.parseReport(this.payload[2]) ??
+				new Duration(0, "unknown");
+
+			this.persistValues();
+		}
+	}
```

Comment:
Lint is unhappy with potentially skipping the call to `persistValues()`.

Error: The application CC report class SceneActuatorConfigurationCCReport needs a call to persistValues() in the constructor or an implementation for mergePartialCCs()

`mergePartialCCs` is not appropriate.  I'm going to move `persistValues()` out of the `if`, so it's always called, and leave the check for `sceneId === 0` in the `persistValues` function (which additionally protects against the possibility that something outside the class calls `persistValues` while `sceneId === 0`).